frontend/server/src/Controllers/Contest.php

```diff
@@ -1666,22 +1667,31 @@ private static function validateCommonCreateOrUpdate(
             'description',
             $isRequired
         );
-
         $r->ensureInt('start_time', null, null, $isRequired);
         $r->ensureInt('finish_time', null, null, $isRequired);
+        $currentStartTime = 0;
+        $currentFinisTime = 0;
```

Comment:
```suggestion
        $currentStartTime = null;
        $currentFinishTime = null;
```

puedes usar `null` en vez de cero en el resto de esta función?

Reply:
Listo